File: ApiAutomovel/Models/Automovel.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ApiAutomovel.Models
{
    public class Automovel
    {
        [Key]
        public int Id { get; set; }
        public string Modelo { get; set; }
        public int NumRodas { get; set; }
        public string Cor { get; set; }
        public string Placa { get; set; }
        public int NumPortas { get; set; }
        public string Descricao { get; set; }
        public decimal Preco { get; set; }       
        public int tipoCarro { get; set; }
    }
}

File: api-crud/src/main/java/com/example/demo/Automovel.java
package com.example.demo;

import java.math.BigDecimal;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Automovel {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	public Long id;
	public String modelo;
	public Integer numRodas;
	public String cor;
	public String Placa;
	public Integer numPortas;
	public String Descricao;
	public BigDecimal preco;
	public String tipoCarro;
	
}
